themes/base/tooltip.css

```diff
@@ -13,8 +13,8 @@
 	position: absolute;
 	z-index: 9999;
 	max-width: 300px;
-	-webkit-box-shadow: 0 0 5px #aaa;
-	box-shadow: 0 0 5px #aaa;
+	-webkit-box-shadow: 0 0 5px #ddd;
+	box-shadow: 0 0 5px #ddd;
```

Comment:
What's the motivation of this change? I'm asking, because it doesn't feel natural to me having a structural stylesheet changed for a theme adjustment.

Perhaps the question must be should we consider moving this into theme.css? Tooltip.css is the only structural stylesheet including a color as far as I can tell.